app/views/todo/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Your To Do List!</h1>
+
+<h4><a href="/todo/new">Add a to do!</a></h4>
```

Comment:
With `resources` the model name would be pluralized (i.e., `/todos/new`). Also strongly encourage you to use path helpers.